tests/e2e/test-build-js.R

```diff
@@ -1,2 +1,30 @@
 rhino::build_js()
-stopifnot(file.exists(fs::path("app", "static", "js", "app.min.js")))
+
+min_js_path <- fs::path("app", "static", "js", "app.min.js")
+index_js_path <- fs::path("app", "js", "index.js")
+
+testthat::expect_equal(
+  readLines(min_js_path, warn = FALSE),
+  "var App;App={};"
+)

```

Comment:
1. Prefer `expect_identical()`, unless you need the relaxed comparison provided by `expect_equal()` ([ref](https://testthat.r-lib.org/reference/equality-expectations.html)). Ditto in other places.
2. Move `build_js()` from the first line right in front of the expectation to make it clear what it checks.

```suggestion
rhino::build_js()
testthat::expect_identical(
  readLines(min_js_path, warn = FALSE),
  "var App;App={};"
)
```